File: Java/Inventario.java
public class Inventario {

  class Objeto {

    String nombre = "";
    int cantidad = 0;

    public Objeto(String nombre) {
      this.nombre = nombre;
    }

    public Boolean actualizarCantidad(int nuevaCantidad) {
      this.cantidad += nuevaCantidad;
    }
  }

  public static void main(String args[]) {
    Objeto botellaDeAgua = new Objeto("Botella de agua");
    botellaDeAgua.actualizarCantidad(10);
    System.out.println(botellaDeAgua.nombre);
  }
}

File: python/inventario.py
class Objeto:

    def __init__(self, nombre):
        self.nombre = nombre
        self.cantidad = 0

    def actualizar_cantidad(self, nueva_cantidad):
        self.cantidad += nueva_cantidad

if __name__ == "__main__":
    botella_de_agua = Objeto(nombre = "Botella de agua")
    botella_de_agua.actualizar_cantidad(nueva_cantidad = 10)
    print(botella_de_agua.nombre)
